cpp/ql/src/Metrics/Classes/CSpecialization.ql

```diff
@@ -3,7 +3,7 @@
  * @description The extent to which a subclass refines the behavior
  *              of its superclasses.
  * @kind treemap
- * @id cpp/specialisation-per-class
+ * @id cpp/specialization-per-class
```

Comment:
This should probably be left as-is, as this will make if difficult for users to compare results produced by different versions of CodeQL.